apis/securesourcemanager/v1alpha1/types.generated.go

```diff
@@ -14,10 +14,6 @@
 
 package v1alpha1
 
-// +kcc:proto=google.cloud.securesourcemanager.v1.Instance.HostConfig
-type Instance_HostConfig struct {
```

Comment:
cc @yuwenma this looks like another example that we need this bug fix.
https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3667